cypress/integration/unit_tests/click_feature_test.js

```diff
@@ -84,7 +82,17 @@ describe('Unit tests for click_feature.js with map and table', () => {
       tableDiv.id = 'table';
       containerDiv.appendChild(tableDiv);
       drawTableAndSetUpHandlers(
-          convertEeObjectToPromise(scoredFeatures).then((fc) => fc.features),
+          convertEeObjectToPromise(scoredFeatures).then((fc) => ({
+                                                          featuresList:
+                                                              fc.features,
+                                                          columnsFound: [
+                                                            geoidTag,
+                                                            blockGroupTag,
+                                                            scoreTag,
+                                                            'OTHER PERCENTAGE',
+                                                            'SOME PROPERTY',
+                                                          ],
+                                                        })),
           map);
```

Comment:
test for these headers to show up somewhere?

Reply:
Good idea, done.